docs/about/release-notes.rst

```diff
@@ -7,7 +7,7 @@ Release Notes
 .. Template, copy this to create a new section after a release:
 
    vRELEASE_PLACEHOLDER
-   --------
+   --------------------
```

Comment:
This should not be edited, the placeholder is replaced before the release.

Reply:
Oh I thought it complained somewhere when I built the docs... maybe not this one... I'll revert it.